modules/acm/src/main/java/edu/gemini/epics/acm/CaWindowStabilizer.java

```diff
@@ -143,4 +147,9 @@ public void removeListener(CaAttributeListener<T> listener) {
         notifier.removeListener(listener);
     }
 
+    public CaWindowStabilizer<T> reset() {
+        filteredVal = lastVal = null;
```

Comment:
It seems like this class relies fairly heavily upon `null`. Could we make it use `Optional<T>` instead?

Reply:
I agree with both points.